pydash/functions.py

```diff
@@ -210,6 +211,33 @@ def callback(item):
         return pyd.some(obj, callback)
 
 
+class flip(object):
```

Comment:
This is not quite how these classes and functions are defined for this submodule. All functions have a corresponding class that implements the function wrapper logic. So there should be a class Flip() and a function flip() where the flip() function returns and instance of the Flip() class. See other function//class pairs for examples.